Source/Pastebin.spoon/init.lua

```diff
@@ -0,0 +1,373 @@
+--- === Pastebin ===
+---
+--- Send clipboard contents to Pastebin
+---
+--- Conversion of tldm's pastebin gist to a Spoon https://gist.github.com/tdlm/5eba0299f2924a8aaf46
+--- Code by @tdlm, spoon by Tyler Thrailkill <tyler.b.thrailkill@gmail.com>
+---
+--- https://github.com/snowe2010
+
+local obj = {}
+obj.__index = obj
+
+-- Metadata
+obj.name = 'Pastebin'
+obj.version = '1.0'
+obj.author = 'Tyler Thrailkill <tyler.b.thrailkill@gmail.com>'
+obj.license = 'MIT - https://opensource.org/licenses/MIT'
+
+--- Pastebin.api_dev_key
+--- Variable
+--- String api developer key. Can be found [here](http://pastebin.com/api)
+obj.api_dev_key = nil
+
+--- Pastebin.api_user_key
+--- Variable
+--- String api user key. Can be generated [here](http://pastebin.com/api/api_user_key.html)
+obj.api_user_key = nil
+
+--- Pastebin.private
+--- Variable
+--- Integer indicating whether a paste should be public, unlisted, or private. Default is 0 (public). (0=public, 1=unlisted, 2=private)
+obj.private = 0
+
+--- Pastebin.expire
+--- Variable
+--- String indicating how long until the paste expires. Default is 'N' (Never)
```

Comment:
I think this probably needs to document the possible values from `expire_times`?